src/mono/msbuild/apple/build/AppleApp.LocalBuild.props

```diff
@@ -23,7 +23,7 @@
 
   <PropertyGroup>
     <_NetCoreAppCurrent>net7.0</_NetCoreAppCurrent>
-    <_NetCoreAppToolCurrent>net6.0</_NetCoreAppToolCurrent>
+    <_NetCoreAppToolCurrent>net7.0</_NetCoreAppToolCurrent>
```

Comment:
@akoeplinger can you please help me understand why this file duplicates these common properties?

Reply:
@ViktorHofer the intention of this file is to be used _outside_ of the dotnet/runtime repo where we don't have access to the common properties (see the description above), i.e. you'd include this .props file in a test .csproj and set RuntimeSrcDir/RuntimeConfig and use your locally built runtime.

We could probably generate this file so we could inject the values for _NetCoreAppCurrent/_NetCoreAppToolCurrent

Reply:
Generating this file would be helpful to avoid multiple sources of truth for the tfm and the version properties.

Reply:
Can you please file an issue?

Reply:
https://github.com/dotnet/runtime/issues/75370